symbols/arm9.yml

```diff
@@ -5016,21 +5087,24 @@ arm9:
         Implements SPECIAL_PROC_INIT_MAIN_TEAM_AFTER_QUIZ (see ScriptSpecialProcessCall).
         
         No params.
-    - name: ScriptSpecialProcess0x3
+    - name: InitSpecialEpisodePartners
       address:
         EU: 0x2048D28
         NA: 0x2048A0C
         JP: 0x2048D78
       description: |-
-        Implements SPECIAL_PROC_0x3 (see ScriptSpecialProcessCall).
+        Initializes the partners for the current special episode and sets the team to be the hero and partner only.
+        Implements SPECIAL_PROC_0x3 (see ScriptSpecialProcessCall). 
         
         No params.
-    - name: ScriptSpecialProcess0x4
+    - name: InitSpecialEpisodeExtraPartner
```

Comment:
I also wasn't sure about this naming convention, as it's really only used to add Celebi to the team in the last special episode. InitSpecialEpisodePartners will add Gardevoir and Medicham for the Team Charm episode so it's not for adding the secondary partner. It has a switch statement to accommodate all the special episodes, so I thought calling it something like AddCelebiInSpecialEpisode was not accurate.